draft/2022-11-30-this-week-in-rust.md

```diff
@@ -109,6 +103,8 @@ Rusty Events between 2022-11-30 - 2022-12-28 🦀
     * [**Rust Hack and Learn**](https://berline.rs/2022/12/06/rust-hack-and-learn.html)
 * 2022-12-06 | Virtual (Buffalo, NY, US) | [Buffalo Rust Meetup](https://www.meetup.com/buffalo-rust-meetup/)
     * [**Buffalo Rust User Group, First Tuesdays**](https://www.meetup.com/buffalo-rust-meetup/events/hlgvxsydcqbjb/)
+* 2022-12-07 | Virtual (Dallas, TX, US) | [Dallas Rust](https://www.meetup.com/Dallas-Rust/)
```

Comment:
It looks like this got moved to 12/13:
https://www.meetup.com/dallas-rust/events/vndgwsydcqbrb/

Reply:
Good eye -- moved